### DISCUSSION TITLE
α 粒子与 β 粒子入射大作业

### DISCUSSION THREAD

**Discussion by liuhanqing-phy** [physics-data/faq#163] [Q&A] [answered]
light_spread.py部分
文字说明是“该部分需要输出光子到达 PMT 的时间和光强”，但给的表是输出光子数。由第一段“发射的所有光子均能到达光电倍增管所在的球面”可知光子数与光强无关，所以这个地方到底要输出啥?
![image](https://user-images.githubusercontent.com/86958697/132853727-3e19fd93-2b05-41fa-81a1-b7116d53a2cc.png)

**Comment by heroxbd** [comment, marked as answer]
@wengjun232

**Reply by wengjun232** [reply]
你给出光子数就行。
